### PR TITLE
drop support for configuring Symfony with Composer 1

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 6.0
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | 
| License       | MIT
| Doc PR        |

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php [hunk 1]
```diff
@@ -235,7 +235,7 @@ class FrameworkExtension extends Extension
     public function load(array $configs, ContainerBuilder $container)
     {
         if (!class_exists(InstalledVersions::class)) {
-            trigger_deprecation('symfony/framework-bundle', '5.4', 'Configuring Symfony without the Composer Runtime API is deprecated. Consider upgrading to Composer 2.');
+            throw new LogicException('The Composer Runtime API is required to configure Symfony. Consider upgrading to Composer 2.');
```
Comment (noniagriconomie) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php]: As now it throws ex, maybe the word `Consider` is no more appropriate, but more `You must upgrade` no?

Comment (nicolas-grekas) [reply src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php]: we usually use "should", so "consider" is fine to me

Comment (stof) [reply src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php]: This should be done using requirements on `composer-runtime-api` in the composer.json, which will then mean that this exception is never triggered...

**Comment by xabbuh** [comment]
I missed that we already have #43788 for this

**Pull request closed without merging**